Cooper/appsettings.json

```diff
@@ -17,7 +17,7 @@
 
   "GmailProvider": {
     "From": "cooper.noreply@gmail.com",
-    "Password": "INSERT_GMAIL_PASSWORD"
+    "Password": "166ADF7CB43FC4D37EE98226D117B953BCF79516"
```

Comment:
"ooops, I did it again") Stop doing this) Yaroslav will not be proud of you

Reply:
Sorry, i'm silly :( I already changed password